tests/urlpatterns/path_urls.py

```diff
@@ -11,7 +11,7 @@
     path('users/', views.empty_view, name='users'),
     path('users/<id>/', views.empty_view, name='user-with-id'),
     path('included_urls/', include('urlpatterns.included_urls')),
-    re_path(r'^regex/(?P<pk>[0-9]+)/$', views.empty_view, name='regex'),
+    re_path(r'^regex/(?P<pk>[0-9]+)/', views.empty_view, name='regex'),
```

Comment:
We can also import `include()` from `django.urls` instead of `django.conf.urls`.